packages/blade/src/utils/storybook/Sandbox/baseCode.ts

```diff
@@ -74,6 +75,7 @@ const getViteReactTSDependencies = (): Dependencies => {
       'styled-components': packageJson.peerDependencies['styled-components'],
       '@emotion/react': '11.11.1',
       '@table-library/react-table-library': '4.1.7',
+      '@razorpay/i18nify-js': '1.4.0',
```

Comment:
we need `@razorpay/i18nify-react` too right?

Reply:
Can we separate it into different PR for `@razorpay/i18nify-react` along with its provider and usage in Amount component example? 

Examples are breaking currently so we can merge this first to unblock consumers

Reply:
`@razorpay/i18nify-react` is not required for Blade to work right now. 
It's just an optional peer dependency for state management. 

If we decide to adopt our discussed approach, only then it will be required. @kamleshchandnani